go.mod

```diff
@@ -54,7 +54,7 @@ require (
 	github.com/prometheus/procfs v0.12.0 // indirect
 	go.uber.org/atomic v1.7.0 // indirect
 	go.uber.org/multierr v1.10.0 // indirect
-	golang.org/x/sys v0.16.0 // indirect
+	golang.org/x/sys v0.17.0 // indirect
```

Comment:
this should be in the replace below `golang.org/x/net`

Reply:
this PR will be closed by https://issues.apache.org/jira/browse/YUNIKORN-2416